gamedata/alldefs_post.lua

```diff
@@ -1054,6 +1054,9 @@ function WeaponDef_Post(name, wDef)
 		-- wDef.targetborder = 1.0
 
 		if wDef.weapontype == "Cannon" then
+			-- do not cast shadows on projectiles
+			wDef.castshadow = false
```

Comment:
consider something like "if not model" so it only affects glowing plasma balls (and not, say, shells)

Reply:
Can you be more specific? I'm not too used to tweaking wdefs.

Reply:
Awesome, thank you!

Reply:
ops, its applying to the whole thing